src/instrument/devices/_apstools_data_management.py

```diff
@@ -33,15 +33,12 @@
 import time
 
 from apstools.utils import run_in_thread
-from ophyd import Component
-from ophyd import Device
-from ophyd import Signal
+from ophyd import Component, Device, Signal
```

Comment:
Can we agree to use all import styles to `isort --sl` (single-line imports only)? This style is used in apstools and beamline_training repos.